crates/bevy_render/src/extract_param.rs

```diff
@@ -74,6 +74,17 @@ where
         }
     }
 
+    #[inline]
+    unsafe fn validate_param(
+        state: &Self::State,
+        system_meta: &SystemMeta,
+        world: UnsafeWorldCell,
+    ) -> bool {
+        // SAFETY: Delegated to existing `SystemParam` implementations.
+        unsafe { Res::<MainWorld>::validate_param(&state.main_world_state, system_meta, world) }
```

Comment:
Added, I'd like someone to make sure this is correct, not super familiar with multi-world stuff

Reply:
Hmm, this might actually be tricky.  It looks like `get_param` passes the *main* world into `state.state.get`, so you should probably pass that instead of `world`.  But the only way to get that is `Res::<MainWorld>::get_param`, which requires `&mut Self::State` instead of just `&Self::State`.  

Maybe you should put it back to what you had originally and make a follow-up issue to make `validate_param` work with `Extract` parameters.

Reply:
I'll mark it as TODO and provide default impl of `true`

Reply:
implemented